week-1-ts-ecosystem/w01e04-light-me-up/src/task-2.ts

```diff
@@ -0,0 +1,41 @@
+export class PowerSource {
```

Comment:
Powtórzony kod z `PowerSource` - to powinno mieszkać (klasa) w osobnym dedykowanym pliku.